e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/CreateNodeE2eTest.java

```diff
@@ -120,61 +103,14 @@ void tearDown()
     @SuppressWarnings({"PMD.JUnitTestsShouldIncludeAssert"})
     void testCreateNodeContainingImageFile() throws IOException
     {
-        // given
-        File imageFile = new File("src/test/resources/images/quick.jpg");
-        List<S3Object> initialBucketContent = awsS3Client.listS3Content();
-
-        // when
-        Node createdNode = repositoryNodesClient.createNodeWithContent(PARENT_ID, imageFile);
-
-        // then
-        RetryUtils.retryWithBackoff(() -> {
-            List<S3Object> actualBucketContent = awsS3Client.listS3Content();
-            assertThat(actualBucketContent.size()).isEqualTo(initialBucketContent.size() + 1);
-
-            WireMock.verify(exactly(1), postRequestedFor(urlEqualTo("/presigned-urls")));
-            WireMock.verify(moreThanOrExactly(2), postRequestedFor(urlEqualTo("/ingestion-events"))
-                    .withRequestBody(containing(createdNode.id())));
-        }, INITIAL_DELAY_MS);
+        testCreateNodeContainingImageFileBase();
```

Comment:
Here the same - both test methods could be entirely placed in parent class IMO

Reply:
Yep - thanks for catching that - I totally had that in mind when annotated the classes with `@DisabledIfEnvironmentVariable`/`@EnabledIfEnvironmentVariable`. 
At first try methods were annotated as I thought it wouldn't work on the whole class - now it's definitely worth moving to the parent.